__tests__/app.test.js

```diff
@@ -496,6 +496,26 @@ describe('GET /api/reviews (queries)', ()=>{
     })
 })
 
+describe( 'GET /api/reviews/:review_id (comment count)', ()=>{
+    test('returns status 200 and the reviews object with comment count', ()=>{
```

Comment:
Really like that you've done this as a seperate test rarther than going back and adding it to your previous tests for this endpoint 👍